inc/plugins/hello.php

```diff
@@ -74,7 +74,7 @@ function hello_info()
 	return array(
 		'name'			=> 'Hello World!',
 		'description'	=> $lang->hello_desc,
-		'website'		=> 'http://mybb.com',
+		'website'		=> 'https://mybb.com',
 		'author'		=> 'MyBB Group',
 		'authorsite'	=> 'http://www.mybb.com',
```

Comment:
Good catch, I'll get that one. 

> On 28 Aug 2016, at 14:03, SKmedix <notifications@github.com> wrote:
> 
> In inc/plugins/hello.php:
> 
> > @@ -74,7 +74,7 @@ function hello_info()
> >  	return array(
> >  		'name'			=> 'Hello World!',
> >  		'description'	=> $lang->hello_desc,
> > -		'website'		=> 'http://mybb.com',
> > +		'website'		=> 'https://mybb.com',
> >  		'author'		=> 'MyBB Group',
> >  		'authorsite'	=> 'http://www.mybb.com',
> What about this?
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub, or mute the thread.
>